app/page.tsx

```diff
@@ -30,7 +34,20 @@ export default function Page() {
           </Link>
         </div>
         <div className="flex items-center justify-center p-6 md:w-3/5 md:px-28 md:py-12">
-          {/* Add Hero Images Here */}
+          <Image
+            src="/hero-desktop.png"
+            width={1000}
+            height={760}
+            className="hidden md:block"
+            alt="Screenshots of the dashboard project showing desktop version"
+          />
```

Comment:
It's good practice to set the width and height of your images to avoid layout shift, these should be an aspect ratio identical to the source image.